test/Data/MessagePackSpec.hs

```diff
@@ -241,6 +259,18 @@ spec = do
       property $ \(a :: (Maybe Int)) -> a `shouldBe` mid a
     it "[int]" $
       property $ \(a :: [Int]) -> a `shouldBe` mid a
+    it "vector int" $
+      property $ \(a :: [Int]) ->
+        let v = V.fromList a
```

Comment:
Ok. It's testsuite, I don't think there could possibly be any Overlapping Instances in future.